arbitrum-docs/node-running/running-an-archive-node.mdx

```diff
@@ -1,13 +1,13 @@
 ---
-title: 'How to run an archive node'
+title: 'How to run an Archive node'
```

Comment:
I think we use "archive" in lowercase throughout the rest of the pages. Should we leave it in lowercase?